app/styles/main.scss

```diff
@@ -29,8 +31,9 @@
   padding: 0 0 1rem 0;
 }
 
-.about_label {
-  font-size: 0.7rem;
+.info {
+  font-size: 1.4rem;
   color: #fc6;
   float: right;
+  box-shadow: none;
 }
```

Comment:
Can we also add `cursor: pointer;` to this block? This would ensure the About icon looks clickable.

```suggestion
.info {
  cursor: pointer;
  font-size: 1.4rem;
  color: #fc6;
  float: right;
  box-shadow: none;
}
```

Reply:
Thanks for noticing. done !!